fsw/src/lc_tbl.h

```diff
@@ -139,33 +139,33 @@ typedef struct
 typedef struct
 {
     uint8 WatchResult; /**< \brief Result for the last evaluation of this
-                                   watchpoint (enumerated)                */
+                                   watchpoint (enumerated) */
 
     uint8 Padding[3]; /**< \brief Structure padding */
 
     uint32 CountdownToStale;            /**< \brief Number of LC Sample Actionpoint
                                                     commands still to be processed
-                                                    before WatchResult becomes stale        */
+                                                    before WatchResult becomes stale      */
     uint32 EvaluationCount;             /**< \brief How many times this watchpoint has
-                                                    been evaluated                         */
+                                                    been evaluated                        */
     uint32 FalseToTrueCount;            /**< \brief How many times this watchpoint has
-                                                    transitioned from FALSE to TRUE        */
+                                                    transitioned from FALSE to TRUE       */
     uint32 ConsecutiveTrueCount;        /**< \brief Number of consecutive times this
-                                                    watchpoint has evaluated to TRUE       */
+                                                    watchpoint has evaluated to TRUE      */
     uint32 CumulativeTrueCount;         /**< \brief Total number of times this watchpoint
-                                                    has evaluated to TRUE                  */
-    LC_WRTTransition_t LastFalseToTrue; /**< \brief Last transition from FALSE to TRUE     */
-    LC_WRTTransition_t LastTrueToFalse; /**< \brief Last transition from TRUE to FALSE     */
+                                                    has evaluated to TRUE                 */
+    LC_WRTTransition_t LastFalseToTrue; /**< \brief Last transition from FALSE to TRUE    */
+    LC_WRTTransition_t LastTrueToFalse; /**< \brief Last transition from TRUE to FALSE    */
 } LC_WRTEntry_t;
 
 /**
  *  \brief Actionpoint Results Table (ART) Entry
  */
 typedef struct
 {
-    uint8 ActionResult; /**< \brief Result for the last sample of this
-                                    actionpoint                            */
-    uint8 CurrentState; /**< \brief Current state of this actionpoint      */
+    uint8 ActionResult; /**< \brief Result for the last sample of this actionpoint */
+
+    LC_APState_t CurrentState; /**< \brief Current state of this actionpoint */
```

Comment:
Same here, it needs to be a defined length in this context (`uint8`)

Reply:
Updated.